### DISCUSSION TITLE
Timepicker | Custom minutes increment | Every 15 minutes

### DISCUSSION THREAD

**Discussion by crisdrag** [mdbootstrap/TW-Elements#1958] [Support from community]
Hi, 

I am using Tailwind Elements, in a React-Vite project. I managed to load it, customise it to accept my min and max times, use the clock face, but i need it to increment the minutes every 15 minutes instead of 5, as it currently is doing. 

Is there a way to do this? Or is there a way to do this in the inline version of the time picker?

Many thanks, 
Cristina

**Comment by juujisai** [comment]
Hi! Unfortunately there is no way of changing the increment value for now. We will discuss this in our team
